x-pack/test/cloud_security_posture_functional/pages/cis_integrations/kspm/cis_integration_eks.ts

```diff
@@ -36,13 +36,14 @@ export default function (providerContext: FtrProviderContext) {
     });
 
     // FLAKY: https://github.com/elastic/kibana/issues/186306
-    describe.skip('KSPM EKS Assume Role', async () => {
+    describe('KSPM EKS Assume Role', async () => {
       it('KSPM EKS Assume Role workflow', async () => {
         const roleArn = 'RoleArnTestValue';
         await cisIntegration.clickOptionButton(CIS_EKS_OPTION_TEST_ID);
+        await cisIntegration.inputIntegrationName('-TEST 1');
```

Comment:
I think this might open room for flakiness in the future (as names might colid across different test suits), what about creating a method that inserts random Input Integration name based on uuid?

so we would just use the following:

```js
await cisIntegration.inputRandomIntegrationName();
```